scripts/ingest.py

```diff
@@ -16,7 +16,12 @@
 from parser.schema.base import Document
 from parser.token_func import group_split
 
-dotenv.load_dotenv()
+import boto3
+import botocore
+from boto3.s3.transfer import TransferConfig
+
+# When override=True .env variables take precedence over environment. https://pypi.org/project/python-dotenv/
+dotenv.load_dotenv(override=True)
```

Comment:
Why has this changed? Default logic looks reasonable.

Reply:
Without `override=true`, the environment takes precedence over the .env variables. If a user should change their AWS credentials in the .env file, I wanted the change to take effect immediately. 😀

Reply:
Sorry, I didn't get the justification and the use case.

Reply:
tldr, if override is not set to true, the aws credentials in .env will not update each time the script is run. 

Use case: User wants to change credentials in their env file between script runs.

Reply:
I don't see your clear understanding of this change. I trust the default behaviour is more generally acceptable and in this case as well.
Please provide a clear case or remove this change.